code/style.css

```diff
@@ -1,5 +1,78 @@
 /* Here in the css file you'll write your styling. make sure to link the css file to the html file. If it's linked correctly you should see a light blue background. */
 
 body {
-  background-color: lightblue;
+  background-color: #d9d9d9;
+  margin-top: 20px;
 }
+
+h1 {
+font-family: Impact, Haettenschweiler, 'Arial Narrow Bold', sans-serif;
+text-align: center;
+}
+
+img {
+  opacity: 0.5;
+}
```

Comment:
This makes it a little bit hard to see your beautiful images 😕 Suggestion:

```
img {
  opacity: 0.8;
}

img:hover {
  opacity: 1;
}
```

Reply:
Hover effects are also a nice way to make the user now an element is clickable 🎈

Reply:
This made it look so much better, thank you for input on this one!